community/kernel/src/test/java/org/neo4j/kernel/impl/store/record/SchemaRuleSerializationTest.java

```diff
@@ -127,6 +132,140 @@ public void shouldReturnCorrectLengthForConstraintRules() throws MalformedSchema
         assertCorrectLength( constraintExistsLabel );
     }
 
+    // BACKWARDS COMPATIBILITY
+
+    @Test
+    public void shouldParseIndexRule() throws Exception
+    {
+        assertParseIndexRule( "/////wsAAAAOaW5kZXgtcHJvdmlkZXIAAAAEMjUuMB9bAAACAAABAAAABA==" );
+        assertParseIndexRule( "AAACAAEAAAAOaW5kZXgtcHJvdmlkZXIAAAAEMjUuMAABAAAAAAAAAAQ=" ); // LEGACY
```

Comment:
I would have gone with hexadecimal strings. They're more readable than base64, which is useful for debugging.